### PR TITLE
Configuration changes for self-monitoring

### PR DESCRIPTION
**Pull request by otoolep**

### CODE REVIEW

**Comment by benbjohnson** [comment]
lgtm

File: etc/config.sample.toml [hunk 1]
```diff
@@ -94,10 +94,10 @@ port = 8087
...
-# InfluxDB can store statistics about itself. This is useful for monitoring purposes.
-# This feature is disabled by default, but if enabled, these statistics can be queried
-# as any other data.
-[statistics]
+# InfluxDB can store statistical and diagnostic information about itself. This is useful for
+# monitoring purposes. This feature is disabled by default, but if enabled, these data can be
+# queried like any other data.
+[monitoring]
 enabled = false
 database = "_internal"         # The database to which the data is written.
```
Comment (otoolep) [inline etc/config.sample.toml]: Gotta nuke this.

**Comment by otoolep** [comment]
Seems like a clustering failure during testing. :-( Kicking the build.

**Comment by otoolep** [comment]
Ok, thanks @benbjohnson 

@pauldix -- merging this. Let me know if config block renaming is an issue.

**Pull request merged**